src/TritonRoute/src/gc/FlexGC_main.cpp

```diff
@@ -402,15 +403,20 @@ bool FlexGCWorker::Impl::checkMetalSpacing_prl_hasPolyEdge(
     return false;
   }
 }
-
+string rectToString(gcRect& r) {
```

Comment:
I don't see a caller.

Reply:
It is meant only for debugging purposes, and the debugging code is gone.